### PR TITLE
feat: Add Swap USD values

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**
Add Swap USD values. 

- `formatUSD` function to format the output string.
- Add `fromAmountUSD` and `setFromAmountUSD`
- Add `toAmountUSD` and `setToAmountUSD`

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: src/swap/components/SwapProvider.tsx [hunk 1]
```diff
@@ -272,12 +278,22 @@ export function SwapProvider({
...
           response.to.decimals,
         );
+        const formattedFromAmountUSD = formatTokenAmount(
+          response.fromAmountUSDC,
+          response.to.decimals, // make const = 6?, OR response.to.decimals
+        );
+        const formattedToAmountUSD = formatTokenAmount(
+          response.fromAmountUSDC,
+          response.to.decimals, // make const = 6?, OR response.to.decimals
+        );
```
Comment (cpcramer) [inline src/swap/components/SwapProvider.tsx:288]: Might not need because we have `getRoundedAmount`

**Comment by cpcramer** [comment]
TODO: Balance: 0.01039493 Max is not showing

File: src/swap/components/SwapAmountInput.tsx [hunk 2]
```diff
@@ -64,6 +63,14 @@ export function SwapAmountInput({
   const hasInsufficientBalance =
     type === 'from' && Number(source.balance) < Number(source.amount);
 
+  const formatUSD = (amount: string) => {
+    if (!amount) {
+      return '~$0.00';
+    }
+    const roundedAmount = Number(getRoundedAmount(amount, 2) || 0);
```
Comment (alessey) [inline src/swap/components/SwapAmountInput.tsx:70]: shouldn't getRoundedAmount always return a value?   when would we hit the 0 fallback?

Comment (cpcramer) [reply src/swap/components/SwapAmountInput.tsx:70]: Good callout, updated!

**Review by 0xAlec — APPROVED** [review]

**Review by abcrane123 — APPROVED** [review]

**Pull request merged**